_targets.R

```diff
@@ -2,22 +2,62 @@ library(targets)
 source("1_fetch/src/get_nwis_data.R")
 source("2_process/src/process_and_style.R")
 source("3_visualize/src/plot_timeseries.R")
+tar_option_set(debug = "site_data")
```

Comment:
You should comment this out for now, too.

Reply:
commented out